ci/ciimage/ubuntu-rolling/install.sh

```diff
@@ -12,7 +12,6 @@ pkgs=(
   python3-pip libxml2-dev libxslt1-dev libyaml-dev libjson-glib-dev
   wget unzip
   qt5-qmake qtbase5-dev qtchooser qtbase5-dev-tools clang
-  pkg-config-arm-linux-gnueabihf
```

Comment:
If this is removed in impish, won't that mean that actually trying to use it in the cross test will now fail?

```
gcc-defaults (1.193) experimental; urgency=medium

  * Bump GCC 10 version to 10.3.0.
  * Bump GCC 11 version to 11.2.0.
  * Also don't build the gccgo-doc package when not building gccgo.
    LP: #1562598.
  * d/rules: Factor-out mips* architectures into macros (Yunqiang Su).
  * Stop building the cross pkg-config packages for Ubuntu.
  * Drop references to powerpcspe.

 -- Matthias Klose <doko@debian.org>  Thu, 05 Aug 2021 09:21:17 +0200
```

Hmm...

Reply:
it looks like some other pkg-config-cross packages are still available in impish, just not armv7

Reply:
But yeah, that means that some of our tests are now skipping instead of running.

Reply:
This is exactly why I don't like running multiple distros in CI, it just creates a ton of fragile moving parts

Reply:
It doesn't seem to have actually broken anything... Which is odd

Reply:
I inspected pkg-config_0.29.2-1ubuntu1.diff.gz, the modifications to Ubuntu's pkg-config package. It installs the original `/usr/share/pkg-config-crosswrapper`, and apparently also something called /usr/share/pkg-config-dpkghook which I think somehow automatically creates these wrappers for every output of `dpkg --print-foreign-architectures` in a post-install hook.

Reply:
I really hate the hidden nature of these, BTW. :/

Reply:
So does that mean that there is in fact a pkg-config anyway?

Reply:
I... think so? The most reliable way to actually test this would be to install the entire environment somewhere with shell access and then actually physically check by hand. :(

Reply:
I actually came up with another idea, I'll look at it tomorrow. Just need to compare the output of main and this PR, if the skips are the same then it's fine

Reply:
So the answer is, there are no changes in the number of passes vs skips between this and the main branch on the ubuntu rolling cross build, so I think we're good.